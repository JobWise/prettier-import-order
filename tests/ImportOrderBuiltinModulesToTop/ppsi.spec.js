run_spec(__dirname, ['typescript'], {
    importOrder: ['^[./]'],
    importOrderBuiltinModulesToTop: true,
});
