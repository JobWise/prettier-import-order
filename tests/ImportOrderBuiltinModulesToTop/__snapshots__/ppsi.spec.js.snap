// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`relative-import-with-builtin-substring.ts - typescript-verify: relative-import-with-builtin-substring.ts 1`] = `
import foo from './constants/foo';
import thirdParty from 'third-party';
~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
import thirdParty from "third-party";
import foo from "./constants/foo";

`;
